src/rshim.c

```diff
@@ -29,18 +29,54 @@
 /* RShim timer interval in milliseconds. */
 #define RSHIM_TIMER_INTERVAL 1
 
+/* Intervals to check the locked mode. */
+#define CHECK_LOCKED_MODE_MS      100
+#define CHECK_LOCKED_MODE_TICKS   (CHECK_LOCKED_MODE_MS / RSHIM_TIMER_INTERVAL)
```

Comment:
Maybe RSHIM_LOCK_MODE_CHECK_MS / RSHIM_LOCK_MODE_CHECK_TICK, so naming will be more close to others